consoles/sshVirtsh.pm

```diff
@@ -517,8 +519,8 @@ __END"
 
     # shut down possibly running previous test (just to be sure) - ignore errors
     # just making sure we continue after the command finished
-    $self->run_cmd("virsh $remote_vmm destroy " . $self->name);
-    $self->run_cmd("virsh $remote_vmm undefine --snapshots-metadata " . $self->name);
+    $self->run_cmd("virsh $remote_vmm destroy " . $self->name . ' >/dev/null 2>&1');
```

Comment:
I'm torn here. We want to ignore expected errors - but hiding unexpected errors makes it harder to debug

Reply:
so `grep -v 'failed to get domain' | grep -v 'Domain not found'`?

Reply:
I don't want to ignore errors on other occassions just here. There is also the comment above stating "ignore errors", that's why :)

Reply:
I'd prefer to see the `grep ...` here instead of ignoring everything.

Reply:
done